### ISSUE TITLE
fortran externals work from terminal but not in jupyter-notebook

### ISSUE THREAD

**Issue by adriantompkins** [climlab/climlab#173]
**PROBLEM**

I switched to mamba/conda and did a fresh install of climlab and then updated - this gives me v0.8.1 on my MAC (OSX12.5.1) 

I also did a python3 -m pip install jupyter from within my conda session

I can import climlab fine in a python3 session now, with no warnings, but if I open a jupyter-notebook (from the same base conda session) and do the same thing I get the warnings that the fortran externals are all missing, 

**SOLUTION (please add to documentation)**

From within the conda environment one needs to install ipython

`conda install ipython
`
and then set up the kernal with 

`ipython kernel install --user --name=base`

One this is done, one can choose the base kernal from the drop down "kernal" menu in the notebook

**Comment by brian-rose** [comment]
@adriantompkins I added some brief instructions on installing climlab + Jupyter in a self-contained environment to the docs here:
https://climlab.readthedocs.io/en/latest/installation.html#installing-into-a-self-contained-conda-environment

Take a look and let me know if this would resolve the problem you described above.

**Comment by adriantompkins** [comment]
@brian-rose sorry to be slow, was in the UK a couple of days. Will try this out in class tomorrow on the ICTP desktops, thanks for following up with a better solution.  I'm a real beginner with conda/mamba, I only switched to help me install climlab.  I needed to reinstall all the packages in climalab-test to get it to work...  Also switching the kernal works ok too, I quite like that solution of being able to choose between the conda environments directly in the notebook.
